File: backend/utils/response.py
"""
统一 JSON 响应工具
所有路由蓝图统一引用此模块，消除重复定义
"""
from flask import jsonify


def json_response(code=200, data=None, msg='success'):
    """统一 JSON 返回结构：{ code, data, msg }"""
    return jsonify({'code': code, 'data': data, 'msg': msg})

File: frontend/lib/response.js
/**
 * 统一 JSON 响应：{ code, data, msg }
 */
function jsonResponse(res, code = 200, msg = 'success', data = null) {
  return res.status(code >= 200 && code < 600 ? code : 500).json({ code, data, msg });
}

export { jsonResponse };
